src/lib/modules/Loan/backoffice/LoanList/LoanList.js

```diff
@@ -29,8 +35,10 @@ export default class LoanList extends Component {
 LoanList.propTypes = {
   hits: PropTypes.arrayOf(PropTypes.object),
   renderListEntryElement: PropTypes.func.isRequired,
+  targetBlanc: PropTypes.string,
 };
 
 LoanList.defaultProps = {
   hits: [],
+  targetBlanc: '',
```

Comment:
not a blocker (typo):
```suggestion
  targetBlank: '',
```